app/assets/javascripts/teams.js

```diff
@@ -0,0 +1,44 @@
+$(document).ready(function() {
+  $(".dropdown-button").dropdown();
+
+  loadLinks();
+
+  function loadLinks() {
```

Comment:
You don't need to define functions within the $(document).ready closure, only to call them. Because loadLinks() automatically calls createLink, you can move both of them out. 

This is something that I've been guilty of too. As the code stands it would be almost impossible for you to test individual methods.